src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,85 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Voltage;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.AutoLogOutput;
+import org.littletonrobotics.junction.Logger;
+
+public class Gripper extends SubsystemBase {
+    private static Gripper INSTANCE;
+    private final GripperIO io;
+    private final GripperInputsAutoLogged inputs = GripperIO.inputs;
+
+    @AutoLogOutput private final Mechanism2d mechanism2d = new Mechanism2d(0, 0);
```

Comment:
You need to add an `@AutoLogOutput` for a `Pose3d` that represents the position of the gripper. This is so AdvantageScope will be able to visualize it. @vichik123 knows about that better than me.

Reply:
b16e92f2b9b1a6d1a7355b4c09d4e192cb0e74dd

Reply:
Not sure how to update it

Reply:
nevermind
bce75c91e6d13b897e3c760f78924f8356ca0989